libsrc/telestrat/scrsize.s

```diff
@@ -0,0 +1,19 @@
+;
```

Comment:
This file is an exception to the comment that I wrote earlier about a leading underscore in library source file names.  It should be named "`_scrsize.s`".  It is a target-dependent helper for "`libsrc/conio/scrsize.s`".